lib/common/index.js

```diff
@@ -38,4 +39,9 @@ for (i = 0; i < vmcommonFunctions.length; i++) {
 var predicateFunctions = Object.keys(predicate);
 for (i = 0; i < predicateFunctions.length; i++) {
     exports[predicateFunctions[i]] = predicate[predicateFunctions[i]];
-}
\ No newline at end of file
+}
+
+var markerFunctions = Object.keys(marker);
+for (i = 0; i < markerFunctions.length; i++) {
+    exports[markerFunctions[i]] = marker[markerFunctions[i]];
```

Comment:
We should assert that we are not replacing an existing
(duplicate) export from another file.